WordPress/Classes/ViewRelated/Stats/Extensions/Double+Stats.swift

```diff
@@ -13,6 +19,34 @@ extension Double {
         }
     }
 
+    private var decimalFormatter: NumberFormatter {
+         get {
+             let formatter = Double._decimalFormatter
+             // Show at least one digit after the decimal
+             formatter.minimumFractionDigits = 1
+             return formatter
```

Comment:
If I understand correctly, the purpose of the `_decimalFormatter` is to cache it since it is a heavy object. Another way to do this so that there is no additional `static var` that is potentially **accessible by other routines** is to make it local using a `struct`:

```swift 
    private var decimalFormatter: NumberFormatter {
         get {
            struct Cache {
                static let formatter: NumberFormatter = {
                    let formatter = NumberFormatter()
                    formatter.minimumFractionDigits = 1
                    return formatter
                }()
            }

            return Cache.formatter
         }
     }

```

Not important for this PR, just thought of mentioning my ideas. 🙂